packages/brand-positioning-block/package.json

```diff
@@ -30,7 +30,7 @@
     },
     "dependencies": {
         "@frontify/app-bridge": "3.0.0-beta.38",
-        "@frontify/fondue": "12.0.0-beta.35",
+        "@frontify/fondue": "12.0.0-beta.114",
```

Comment:
I would suggest to not update fondue in all the other blocks. With all the changes in fondue, we need to test the block if it still works as expected. This here is also quite a big step: version 35 to 114.

Reply:
cypress has issues with different versions installed in the packages. the brand positioning block shouldn't have issues with the newest version, but I see your point, it would be nice to have different versions